inscriptions/ISic3002.xml

```diff
@@ -190,25 +195,20 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140306">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#VM">
                 <ab>
-                    <lb n="1"/>
-                    <supplied reason="lost">— — —</supplied>․<supplied reason="lost">— — — — — — —</supplied>#⁷
-                    <lb n="2"/>
-                    <supplied reason="lost">— —</supplied>ΤΗΣ<supplied reason="lost">— — — — — — —</supplied>Υ
-                    <lb n="3"/>
-                    <supplied reason="lost">— —</supplied>άδα τὰν α
-                    <unclear>ὑτ</unclear>
-                    <supplied reason="lost">οῦ γ</supplied>
-                    <unclear>υ</unclear>ναῖκα
-                    <lb n="4"/>
-                    <supplied reason="lost">— —</supplied>
-                    καὶ Ἴσει.
-                    <lb n="5"/>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/><orig cert="low"><unclear>Τ</unclear></orig><gap reason="lost" extent="unknown" unit="character"/><orig><unclear>Ε</unclear></orig>
```

Comment:
not sure about cert=low on orig here? If it's unclear, I think that's sufficient for any (?). The reality is it's just a trace of a vertical (and only the very bottom part of it - i've seen the stone), and so I don't think this should be more than `<gap reason="illegible" unit="character" quantity="1"/>`, which will just give a . in the transcription.